### ISSUE TITLE
Add ggspatial to ggplot2 extensions page

### ISSUE THREAD

**Issue by jl5000** [paleolimbot/ggspatial#35]
Not strictly a code issue, but it would be good if this was registered here:

https://www.ggplot2-exts.org/

**Comment by jl5000** [comment]
The instructions are here: https://github.com/ggplot2-exts/gallery#adding-a-ggplot2-extension

Although, it looks as though the owner is not too active.

**Comment by thiesben** [comment]
Also, the above link now links to porn. You should consider remove it?
Perhaps try to submit ggspatial here: https://exts.ggplot2.tidyverse.org/gallery/! :)

**Comment by jl5000** [comment]
> Also, the above link now links to porn. You might consider to remove it...
> Perhaps try to submit ggspatial here: https://exts.ggplot2.tidyverse.org/gallery/! :)

Yes, good call. I noticed this a couple of weeks ago but completely forgot I had posted it here.
